static/app/utils/useColorscheme.tsx

```diff
@@ -28,7 +28,7 @@ function setFaviconTheme(theme: 'dark' | 'light'): void {
 
 export function useColorscheme() {
   const {user} = useLegacyStore(ConfigStore);
-  const configuredTheme = user?.options.theme ?? 'system';
+  const configuredTheme = user?.options?.theme ?? 'light';
```

Comment:
Default sentry theme is light, so if the user hasn't selected a theme, should default to light.

Reply:
what about this line below: 
```
const theme = configuredTheme === 'system' ? preferredTheme : configuredTheme;
```

Reply:
Yeah I think default is to match system prefs unless overridden